src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_doing_flr_with_native_transactions.cs

```diff
@@ -14,9 +15,16 @@ public void Should_do_5_retries_by_default_with_native_transactions()
             Scenario.Define(() => new Context { Id = Guid.NewGuid() })
                     .WithEndpoint<RetryEndpoint>(b => b.Given((bus, context) => bus.SendLocal(new MessageToBeRetried { Id = context.Id })))
                     .AllowExceptions()
-                    .Done(c => c.ForwardedToErrorQueue || c.NumberOfTimesInvoked > 5)
+                    .Done(c => c.ForwardedToErrorQueue)
                     .Repeat(r => r.For(Transports.Default))
-                    .Should(c => Assert.AreEqual(5+1, c.NumberOfTimesInvoked, "The FLR should by default retry 5 times"))
+                    .Should(c =>
```

Comment:
The point is that we want to move to more AAA style testing and have he Assertions clearly visible on the level of the test method outside the ATT runner. Let's address this outside of this PR and then remove `Should` at least.